File: Codeforces/C++/problemset/Hit the Lottery.cpp
// Problem Link: https://codeforces.com/problemset/problem/996/A

#include <iostream>
using namespace std;
typedef long long ll;
int main()
{
    ll n, x = 0;
    cin >> n;
    x += n/100;
    n = n%100;
    x += n/20;
    n = n%20;
    x += n/10;
    n = n%10;
    x += n/5;
    n = n%5;
    x += n;
    cout << x;
    return 0;
}

File: src/main/kotlin/HitTheLottery.kt
package problems

/* Problem Link: https://codeforces.com/problemset/problem/996/A */

fun main() {
    val n = readln().toInt()
    print((n / 100) + ((n % 100) / 20) + ((n % 20) / 10) + ((n % 10) / 5) + ((n % 5) / 1))
}